### ISSUE TITLE
AttributeError in TrajectoryHOOMD due to hoomd API change

### ISSUE THREAD

**Issue by bdice** [glotzerlab/freud#38]
**[Original report](https://bitbucket.org/glotzer/freud/issue/38) by Carl Simon Adorf (Bitbucket: [csadorf](https://bitbucket.org/csadorf), GitHub: [csadorf](https://github.com/csadorf)).**

----------------------------------------

Error occurs due to API change with hoomd's box dim
```
#!python
  File "/home/simon/.local/lib/python2.7/site-packages/freud/trajectory.py", line 139, in __iter__
    yield self[idx];
  File "/home/simon/.local/lib/python2.7/site-packages/freud/trajectory.py", line 132, in __getitem__
    return self._get_current_frame();
  File "/home/simon/.local/lib/python2.7/site-packages/freud/trajectory.py", line 1075, in _get_current_frame
    box = Box(hoomd_box[0], hoomd_box[1], hoomd_box[2], self.sysdef.dimensions == 2);
AttributeError: boxdim instance has no attribute '__getitem__'
```
I fixed it locally and will send a pull request immediately.

**Comment by bdice** [comment]
**Original comment by Carl Simon Adorf (Bitbucket: [csadorf](https://bitbucket.org/csadorf), GitHub: [csadorf](https://github.com/csadorf)).**

----------------------------------------

Fix Issue #38.

**Comment by bdice** [comment]
**Original comment by Eric Harper (Bitbucket: [harperic](https://bitbucket.org/harperic), GitHub: [harperic](https://github.com/harperic)).**

----------------------------------------

Merged in issue38 (pull request #29)

Fix Issue #38.
